structurizr/workspace.dsl

```diff
@@ -0,0 +1,260 @@
+							}
+
+							containerFoAzureSql = container "Azure SQL Database" "Data store for appeals" "Azure SQL" {
+								tags "Microsoft Azure - SQL Database" "Database"
+							}
+
+							containerFoAzureCosmos = container "Azure Cosmos Mongo Database" "Data store for appeals" "Azure Cosmos" {
+								tags "Microsoft Azure - Azure Cosmos DB" "Database"
+							}
+						}
+
+						group "Appeal Front-Office APIs" {
+							containerFoApi = container "Appeal Front-Office API" "API" "Node.js, Azure Web App" {
+								tags "Microsoft Azure - App Services"
+							}
+
+							containerFoDocsApi = container "Appeal Front-Office Document API" "API" "Node.js, Azure Web App" {
+								tags "Microsoft Azure - App Services"
+							}
+
+							containerFoPdfApi = container "Appeal Front-Office PDF API" "API" "Node.js, Azure Web App" {
+								tags "Microsoft Azure - App Services"
+							}
+						}
+
+						containerFoClamAV = container "ClamAV" {
+							tags "Microsoft Azure - Container Instances"
+						}
+
+						containerFoAuthServer = container "Appeal Front-Office Auth Server" "API" "Node.js, Azure Web App" {
+							tags "Microsoft Azure - App Services"
+						}
+
+						containerFunctionApp = container "integration-functions" "Consumes messages from Service Bus" "Function App, JavaScript" {
+							tags "Microsoft Azure - Function Apps", "FunctionApp"
+						}
+					}			
+				}
+			}
+
+			systemAppsBo = softwareSystem "Appeals Back-Office" "Internal service to manage planning appeals in England"  {
+				// simplified, see equivalent diagram in back office for more detail
+				group "Appeal Back-Office VNET" {
+					containerBoFileStorage = container "Storage Account" "Document storage" {
+						tags "Microsoft Azure - Storage Accounts"
+					}
+
+					containerBoWeb = container "Appeal Back-Office Web" "Web UI" "Node.js, Azure Web App" {
+						tags "Browser,Microsoft Azure - App Services"
+					}	
+				}
+			}
+
+			systemIntegration = softwareSystem "Integration Layer" "Data exchange between FO and back office" {
+				containerBoServiceBus = container "Service Bus" "Messaging platform for data exchange" "Azure ServiceBus Namespace" {
+					tags "Microsoft Azure - Azure Service Bus"
+
+					group "Commands" {
+						componentCaseCmdTopic = component "Service Bus Appeal submission topic" "Appeal submission commands"
+						componentLpaqCmdTopic = component "Service Bus LPA submission topic" "LPA submission commands"
+					}
+
+					group "Events" {
+						componentCaseMessageTopic = component "Service Bus Appeal topic" "Appeal messages"
+						componentDocumentMessageTopic = component "Service Bus Document topic" "Appeal document messages"
+						componentServiceUserMessageTopic = component "Service Bus Service User topic" "Service user messages"
+						componentEventMessageTopic = component "Service Bus Event topic" "Event message"
+					}
+				}
+			}
+
+            systemGovUk = softwareSystem "GOV Notify" {
```

Comment:
This block is sitting within the "Planning Inspectorate: Appeal" group, which means on the system landscape diagram it's showing incorrectly as sitting within the Planning Inspectorate

http://localhost:8082/workspace/diagrams#SystemLandscape-001

You'll need to move this block down to line 143 to resolve